pkg/strategy/controller.go

```diff
@@ -29,29 +30,37 @@ var (
 
 // For returns the appropriate Strategy for the given Jaeger instance
 func For(ctx context.Context, jaeger *v1.Jaeger, secrets []corev1.Secret) S {
+	tracer := global.TraceProvider().GetTracer(v1.ReconciliationTracer)
+	ctx, span := tracer.Start(ctx, "strategy")
```

Comment:
Is the `strategy` name appropriate? For the others it is the function name - but `For` is also not ideal. Sorry no suggestions.

Reply:
`strategyFor`?

Reply:
`strategy.For` +1, or if agreeing to the scoping of operation names, then `strategy.controller.For`?